apps/src/templates/progress/ProgressBubbleSet.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * An ordered set of progress bubbles.
+ */
+
+import React from 'react';
+import Radium from 'radium';
+import ProgressBubble from './ProgressBubble';
+import color from "@cdo/apps/util/color";
+
+const styles = {
+  main: {
+    position: 'relative',
+    display: 'inline-block'
+  },
+  withBackground: {
+    display: 'inline-block',
+    position: 'relative'
+  },
+  background: {
+    height: 10,
+    backgroundColor: color.lighter_gray,
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    marginTop: (ProgressBubble.height - 10) / 2,
+    marginBottom: (ProgressBubble.height - 10) / 2,
+  },
+  backgroundFirst: {
+    left: 15
+  },
+  backgroundLast: {
+    right: 15
+  },
+  bubble: {
+    position: 'relative',
+  }
+};
+
+const ProgressBubbleSet = React.createClass({
+  propTypes: {
+    startingNumber: React.PropTypes.number.isRequired,
```

Comment:
Deprecated, but I don't see any indication that it was ever actually removed or is unsupported anywhere. Anyway, you wouldn't be depending on it - just making a naming suggestion.